test/license_parser_test.dart

```diff
@@ -7,63 +7,16 @@ void main() {
   group("Parsing Versions of the AAMVA DL/ID Card Design Standard", () {
     group("Parsed License", () {
       test("should expose the raw pdf417 data", () {
-        final rawData = "@\n" +
-            "\n" +
-            "ANSI 636026010102DL00410288ZA03290015DLDBJD12345678\n" +
-            "DAAPUBLIC,JOHN,Q\n" +
-            "DABPUBLIC\n" +
-            "DACJOHN\n" +
-            "DADQUINCY\n" +
-            "DAG789 E OAK ST\n" +
-            "DAIANYTOWN\n" +
-            "DAHAPT #4A\n" +
-            "DAJCA\n" +
-            "DAK90223\n" +
-            "DBB19700115\n" +
-            "DBA20350131\n" +
-            "DAU509\n" +
-            "DAW180\n" +
-            "DAZBR\n" +
-            "DAYGRN\n" +
-            "DBC1\n" +
-            "DBHN\n" +
-            "DARD\n" +
-            "DBD20131003\n" +
-            "DBE2013-10-03 00.00.00.000000\n" +
-            "DBIN\n" +
-            "ZAZAA7V81\n";
+        final rawData =
+            "@\n\nANSI 636026010102DL00410288ZA03290015DLDBJD12345678\nDAAPUBLIC,JOHN,Q\nDABPUBLIC\nDACJOHN\nDADQUINCY\nDAG789 E OAK ST\nDAIANYTOWN\nDAHAPT #4A\nDAJCA\nDAK90223\nDBB19700115\nDBA20350131\nDAU509\nDAW180\nDAZBR\nDAYGRN\nDBC1\nDBHN\nDARD\nDBD20131003\nDBE2013-10-03 00.00.00.000000\nDBIN\nZAZAA7V81\n";
```

Comment:
@moesaid I think I actually prefer these test strings the way they were, broken down line by line. It makes it easier to inspect the raw data. Can you revert these? If it is running afoul of the analyzer, please ignore that rule so that our test cases can be more clear.

Reply:
you got it!